src/components/member/PasswordSetting.js

```diff
@@ -0,0 +1,228 @@
+          >
+            <Grid item xs={12} sm={6}>
+              {user.get('password') !== null ? (
+                <TextField
+                  className={classes.input}
+                  required
+                  label={t('Current Password')}
+                  variant="outlined"
+                  value={currentPassword}
+                  error={currentPasswordError}
+                  helperText={currentPasswordError}
+                  onChange={handleCurrentPasswordInput}
+                  id={USER_CURRENT_PASSWORD_INPUT_ID}
+                  type="password"
+                />
+              ) : (
+                <Grid item xs={12} sm={6} />
+              )}
+            </Grid>
+            <Grid item xs={12} sm={6} />
+            <Grid item xs={12} sm={6}>
+              <TextField
+                className={classes.input}
+                required
+                label={t('New Password')}
+                variant="outlined"
+                value={newPassword}
+                error={newPasswordError}
+                helperText={newPasswordError}
+                onChange={handleNewPasswordInput}
+                id={USER_NEW_PASSWORD_INPUT_ID}
+                type="password"
+              />
+            </Grid>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                className={classes.input}
+                required
+                label={t('Confirm Password')}
+                variant="outlined"
+                value={confirmPassword}
+                error={confirmPasswordError}
+                helperText={confirmPasswordError}
+                onChange={handleConfirmPasswordInput}
+                id={USER_CONFIRM_PASSWORD_INPUT_ID}
+                type="password"
+              />
+            </Grid>
+            <Grid container direction="row">
+              <Grid item xs={12} sm={6} className={classes.buttonItem}>
+                <Typography className={classes.note}>
+                  {t(
+                    'Make sure it is at least 8 characters including a number, a lowercase letter and an uppercase letter.',
+                  )}
+                </Typography>
+                <Button
+                  id={CONFIRM_CHANGE_PASSWORD_BUTTON_ID}
+                  variant="contained"
+                  color="primary"
+                  onClick={() => handleChangePassword()}
+                >
+                  {t('Update password')}
+                </Button>
+              </Grid>
+              <Grid item xs={12} sm={6} className={classes.buttonItem}>
+                <Typography className={classes.note}>
+                  {t('I forgot my password:')}
+                </Typography>
+                <Button
+                  id={CONFIRM_RESET_PASSWORD_BUTTON_ID}
+                  variant="outlined"
+                  disabled
+                  // TO DO:
+                  // onClick={() => handleChangePassword()}
```

Comment:
Change the code? It was supposed to reset the password right?

Reply:
The idea is to request a password reset to the admin, which will be handled from the Admin panel. I leave it as work to do